app/views/chatrooms/edit.html.slim

```diff
@@ -0,0 +1,9 @@
+h1
+ = t ".title.edit"
+
+== render 'form'
+
+=> link_to t(".btn-show"), @chatroom
+'|
+=< link_to t(".btn-back"), chatrooms_path
```

Comment:
what the hell `==` || `=>` || `=<`

Reply:
cái này nó tự sinh ra, sau này cũng sửa lại form này mà

Reply:
thì sửa luôn lần a ơi, sửa cho quen :))